cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/PushUtils.java

```diff
@@ -98,11 +101,19 @@ public static void pushPackages(String packageName, String sourceRoot, String in
         Path pkgPathFromPrjtDir = Paths.get(prjDirPath.toString(), ProjectDirConstants.DOT_BALLERINA_DIR_NAME,
                                             ProjectDirConstants.DOT_BALLERINA_REPO_DIR_NAME, orgName,
                                             packageName, version, packageName + ".zip");
-        if (Files.notExists(pkgPathFromPrjtDir)) {
-            BuilderUtils.compileWithTestsAndWrite(prjDirPath, packageName, packageName, false, false, false, true);
-        }
+        // Build the package
+        BuilderUtils.compileWithTestsAndWrite(prjDirPath, packageName, packageName, false, false, false, false);
 
         if (installToRepo == null) {
+            if (!pushToRepo) {
+                outStream.print("Push the artifact " + orgName + "/" + packageName + ":" + version  + " to central " +
+                                        "[yes/y, no/n]: (y) ");
+                Scanner scanner = new Scanner(System.in, Charset.defaultCharset().name());
+                String buildOnPush = scanner.nextLine().trim();
+                if (buildOnPush.equalsIgnoreCase("no") || buildOnPush.equalsIgnoreCase("n")) {
```

Comment:
If some one say anything other than no/n , this will get installed right? If yes, shall we check for the yes/y and proceed?

Reply:
Added the check with 3b98edc